executor/benches/bench.rs

```diff
@@ -1,235 +0,0 @@
-// This file is part of Substrate.
-
-// Copyright (C) 2018-2021 Parity Technologies (UK) Ltd.
-// SPDX-License-Identifier: Apache-2.0
-
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// 	http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-use codec::{Decode, Encode};
```

Comment:
what the use of this file ? why delete it

Reply:
This file is out of date, it can't build.
```rust
use node_executor::Executor;
```
This `Executor` doesn't exist in `executor/src/lib.rs`, but bench.rs uses it.
Running `cargo bench` will report some errors.
![image](https://user-images.githubusercontent.com/4054836/147740892-752b5440-362e-4d83-9d72-2ba765ed6b13.png)

Reply:
maybe another pr to fix this. this caused by use new substrate

Reply:
Yes, we can discuss how to do proper benchmark of deeper-chain in the future. This benchmark looks like to test how fast the executor can construct block.

Reply:
I fixed this benchmark, it now can run